packages/http-client-csharp/emitter/src/options.ts

```diff
@@ -3,7 +3,7 @@ import { EmitContext, JSONSchemaType, resolvePath } from "@typespec/compiler";
 import { tspOutputFileName } from "./constants.js";
 import { LoggerLevel } from "./lib/log-level.js";
 
-export type NetEmitterOptions = {
+export interface NetEmitterOptions extends SdkEmitterOptions {
```

Comment:
change NetEmitterOptions to interface, so that

- it can be extended; 
- consistent with others e.g. SdkEmitterOptions;
-  it can used as partial scenario to add more fields which may more suitable for plugin framework.